src/graph/visitor/PrunePropertiesVisitor.cpp

```diff
@@ -57,7 +57,7 @@ void PrunePropertiesVisitor::visitCurrent(Project *node) {
     rootNode_ = false;
     return;
   }
-  for (auto i = 0u; i < columns.size(); ++i) {
```

Comment:
This don't fix the bug.

Reply:
The correctness is guaranteed by constraint which size of `colName` is equal to `columns`, or some unexpected error will be reported.

Reply:
This constraint may not be valid. There may be valid columns without names. And, the current design in general does not enforce this. These two are different contatiners. So I think it's an issue larger than this crash bug. I opened a new issue to track this https://github.com/vesoft-inc/nebula/issues/4881.

Reply:
If we want to enforce their sizes to be the same, better to organize them as AoS. If necessary, that better be left as another issue to fix.

Reply:
> If we want to enforce their sizes to be the same, better to organize them as AoS. If necessary, it better be left as another issue to fix.

Ignore this, checked, not feasible.